docs/Legacy ChangeLog.txt

```diff
@@ -0,0 +1,323 @@
+Minicraft+ version 2.2.0
```

Comment:
Put this back in the original changelog

Reply:
Why? I don't understand.

Reply:
Leave the old changelog in the file as a quote or something.

Like this:
```
Minicraft+ version 2.0.7

+ Added sheep shearing
+ Added shear tool
+ Added potatoes
+ Added baked potatoes as food
+ Made zombies drop potatoes
+ Added inventory searcher (F to toggle and Next/Back Page to seek)
```

Reply:
Remove this file as well.